docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+  web:
+    build: .
+    ports:
+      - "5001:5000"
+    depends_on:
+      - db
+    env_file:
+      - .env.docker
+    volumes:
+      - .:/app
+    restart: always
+    # Add host entry for the db service
+    extra_hosts:
+      - "db:172.17.0.1"
+
+  db:
+    image: postgres:17
```

Comment:
Not sure about the Postgres version being used in production these days at nyc-records, but this version worked with the app. I did see some references to earlier versions of Postgres (9.something?) but I couldn't get that to work